doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -233,41 +233,41 @@ Surface Tension Force
       When the surface tension force is enabled, a ``fluid-fluid`` material interaction and a ``surface tension model`` with its parameters must be specified in the :doc:`physical_properties` subsection.
 
   * ``verbosity``: enables the display of the output from the surface tension force calculations. Choices are: ``quiet`` (default, no output) and ``verbose``.
-  * ``output auxiliary fields``: enables the display of the filtered ``phase fraction gradient`` and filtered ``curvature``. Used for debugging purposes.
+  * ``output auxiliary fields``: enables the display of the projected ``phase fraction gradient`` and projected ``curvature``. Used for debugging purposes.
 
-  * ``phase fraction gradient filter factor``: value of the factor :math:`\alpha` applied in the filter :math:`\eta_n = \alpha h^2`, where :math:`h` is the cell size. This filter is used to apply a `projection step <https://onlinelibrary.wiley.com/doi/full/10.1002/fld.2643>`_ to damp high frequency errors, that are magnified by differentiation, in the phase fraction gradient (:math:`\bf{\psi}`), following the equation:
+  * ``phase fraction gradient diffusion factor``: value of the factor :math:`\alpha` in :math:`\eta_n = \alpha h^2`, where :math:`h` is the cell size. This diffusion term is used in a `projection step <https://onlinelibrary.wiley.com/doi/full/10.1002/fld.2643>`_ to damp high frequency errors, that are magnified by differentiation, in the phase fraction gradient (:math:`\bf{\psi}`), following the equation:
```

Comment:
Since we do not explicit about the whole "term" here, I would say "coefficient" here.

```suggestion
  * ``phase fraction gradient diffusion factor``: value of the factor :math:`\alpha` in :math:`\eta_n = \alpha h^2`, where :math:`h` is the cell size. This diffusion coefficient (:math:`\eta_n`) is used in a `projection step <https://onlinelibrary.wiley.com/doi/full/10.1002/fld.2643>`_ to damp high frequency errors, that are magnified by differentiation, in the phase fraction gradient (:math:`\bf{\psi}`), following the equation:
```